addons/im_livechat/static/tests/tours/chatbot_step_type_clear_only.js

```diff
@@ -32,7 +32,8 @@ registry.category("web_tour.tours").add("change_chatbot_step_type", {
             run: "click",
         },
         {
-            trigger: ".o_form_view",
+            // Ensure form is properly saved, in which case the save button is hidden.
+            trigger: ".o_form_status_indicator_buttons:not(:visible)",
```

Comment:
Shouldn't trigger be not o_form_button_save or o_form_saved or something like that?